File: app/src/main/java/by/d1makrat/library_fm/ui/fragment/tabTop/TabTopArtistsFragment.java
package by.d1makrat.library_fm.ui.fragment.tabTop;

import by.d1makrat.library_fm.adapter.pages.TabTopArtistsAdapter;

public class TabTopArtistsFragment extends TabTopItemsFragment<TabTopArtistsAdapter> {

    @Override
    protected TabTopArtistsAdapter createAdapter() {
        return new TabTopArtistsAdapter(getChildFragmentManager());
    }
}

File: app/src/main/java/by/d1makrat/library_fm/ui/fragment/tabTop/TabTopArtistsFragment.kt
package by.d1makrat.library_fm.ui.fragment.tabTop

import by.d1makrat.library_fm.adapter.pages.TabTopArtistsAdapter

class TabTopArtistsFragment : TabTopItemsFragment<TabTopArtistsAdapter>() {

    override fun createAdapter(): TabTopArtistsAdapter {
        return TabTopArtistsAdapter(childFragmentManager)
    }
}
